### ISSUE TITLE
Focus style for buttons and links should have yellow background (React)

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#820]
See how this is done in gov uk design system: https://design-system.service.gov.uk/get-started/focus-states/

**Comment by mrosvik** [comment]
Blocked by https://github.com/digdir/designsystem/issues/833
